environments/frontend.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+	extends: 'zipscene/core',
+	env: {
+		'shared-node-browser': false
```

Comment:
I'm on the fence here. If you're disabling the shared-node-browser in this file, you should specify the environment (`browser`), which you aren't doing until the next child.